README.md

```diff
@@ -1 +1,63 @@
-# FunnySectors
\ No newline at end of file
+
+# Funny Sectors
+
+A lightweight plugin for managing large minecraft networks
+
+
+## Important note
+
+We do not have a working demo yet. [we](https://github.com/FunnyGuilds/FunnySectors/graphs/contributors) are actively working on creating one.
+
+
+## FAQ
+
+#### What are the priorities for the project?
+
+Well... We need a working demo. That is the most important thing for now. As [bopke](https://github.com/bopke) (project leader) [mentioned](https://github.com/FunnyGuilds/FunnySectors/issues/6) we need a working demo to avoid burnout.
+
+#### What technology are you going to start with?
+
+* Project language: [Java](https://www.java.com/en/)
+* Test language: [Kotlin](https://kotlinlang.org/)
+* Database: [MySql](https://www.mysql.com/)
```

Comment:
Thread about database wasn't concluded and in fact, I really oppose using mysql as our database. I believe that especially for our case, going with schemaless database like mongo would be way easier to iterate.

Reply:
> Thread about database wasn't concluded

At the time of writing this non of the threads were close so... if you have any opinions please writhe it in the issues section and when we come to a conclusion I will change it

Reply:
```suggestion
```
Let's add this information when we conclude database selection then